packages/HorizonUI/Sources/HorizonUI/Sources/Components/Button/HorizonUI.ButtonStyles.swift

```diff
@@ -57,108 +99,155 @@ extension HorizonUI {
                     .foregroundColor(foreground)
             }
             .huiTypography(.buttonTextLarge)
-            .tracking(100)
-            .padding(.horizontal, 16)
+            .padding(.horizontal, .huiSpaces.primitives.mediumSmall)
             .frame(height: isSmall ? 40 : 44)
             .frame(maxWidth: fillsWidth ? .infinity : nil)
             .background(background)
             .foregroundStyle(foreground)
             .cornerRadius(isSmall ? 20 : 22)
             .opacity(isEnabled ? (configuration.isPressed ? 0.8 : 1.0) : 0.5)
         }
+
+        private func makeIconOnlyButtonType(configuration: Configuration) -> any View {
+            guard let icon = icon else {
+                return EmptyView()
+            }
+            return ZStack {
+                icon
+                    .renderingMode(.template)
+                    .frame(width: isSmall ? 40 : 44, height: isSmall ? 40 : 44)
+                    .background(background)
```

Comment:
I have seen icon size is repeated more than once, please set the values in property.
`.renderingMode(.template)` is not need for now all images are in the assets are template, 
And please change the code to be like this 

```
@ViewBuilder
        private func makeIconButtonType(icon: Image, configuration: Configuration) -> some View {
            ZStack {
                icon
                    .frame(width: isSmall ? 40 : 44, height: isSmall ? 40 : 44)
                    .background(background)
                    .foregroundStyle(foreground)
                    .huiCornerRadius(level: .level6)
                    .foregroundColor(foreground)
                    .opacity(isEnabled ? (configuration.isPressed ? 0.8 : 1.0) : 0.5)

                if let badge = badge, let badgeStyle = badgeStyle {
                    HorizonUI.Badge(type: .number(badge), style: badgeStyle)
                        .offset(x: 15, y: -15)
                }
            }
        }
```